README.md

```diff
@@ -59,9 +58,9 @@ curl -G http://localhost:8086/query?pretty=true --data-urlencode "db=mydb" \
 
 ## Documentation
 
-* Read more about the [design goals and motivations of the project](https://docs.influxdata.com/influxdb/v0.10/).
-* Follow the [getting started guide](https://docs.influxdata.com/influxdb/v0.10/introduction/getting_started/) to learn the basics in just a few minutes.
-* Learn more about [InfluxDB's key concepts](https://docs.influxdata.com/influxdb/v0.10/guides/writing_data/).
+* Read more about the [design goals and motivations of the project](https://docs.influxdata.com/influxdb/v0.12/).
```

Comment:
We should switch all these links to use `https://docs.influxdata.com/influxdb/latest/`